docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/01_instrument_intro_prerequisites.md

```diff
@@ -0,0 +1,129 @@
+
+## Prerequisites
+
+1. Prepare your **Case Report Forms (CRFs)** including any “look-up” tables used for 
+scoring/normalization and any documentation defining possible relationships between 
+fields for validation.
+2. Determine which **sections of the questionnaire should be included or omitted** 
+from the online version. Typically, unless they pertain to the patients themselves, 
+administrative details included on the original form can be ignored, but instructions 
+that will be helpful for the examiner/patient might be included. 
+3. Decide **how all your fields would map to the form element** types allowed by LORIS
+(Refer to the Instrument Builder module to get a list of field types available for LORIS forms)
+4. Determine if your instrument requires **special certifications** to be administered. 
+If so, only certified examiners will be able to sign the instrument. (*For more 
+information, see [Additional Configurations - Certification](04_instrument_additional_configurations.md#Certification) section*)
+5. Determine if your instrument will have **special access restrictions** other than 
+LORIS users from the same site and the same project. LORIS allows for instrument 
+specific access permissions if needed. (*For more information, see [Additional Configurations - Permissions](04_instrument_additional_configurations.md#Permissions) section*)
+6. Determine if your instrument will be **administered by an examiner** and Data-entry 
+will be done by personnel **OR** if the instrument will be completed by the **candidate 
+directly**.(*For more information, see [Additional Configurations - Surveys](04_instrument_additional_configurations.md#Surveys) section*)
+7. Decide which **Quality Assurance (QA)** features you would like to use for the 
+instrument [Double Data Entry](05_instrument_quality_assurance.md#Double-Data-Entry) 
+and/or [Validity Flagging](05_instrument_quality_assurance.md#Validity)
+
+
+## Instrument Development Workflow
+
+_This section is meant to describe the recommended workflow of developing and 
+installing an instrument in the database. The next sections in the documentation 
+follow the same order as the steps listed below._
+
+1. Read the entire "instrument" documentation before getting started.
+2. Determine your answer for every in the [Prerequisites](#Prerequisites) section above
+3. Start Developing your instrument either using the [**LINST**](./02_instrument_format/LINST_instrument.md) 
+or [**PHP**](./02_instrument_format/PHP_instrument.md) format.
+4. Install your instrument by following the [Installation Instructions](03_instrument_install.md)
+5. Configure your new instruments following the [Additional Configurations](04_instrument_additional_configurations.md) section
+6. Finally, [set up your behavioural test battery](07_clinical_test_battery.md) in 
+7. order to be able to [test and troubleshoot](06_instrument_testing_and_troubleshooting.md) your instrument 
```

Comment:
The "in order" being split between 6/7 flows really weirdly.

Reply:
I'll try to amend

Reply:
it was just a wrong order formatting